quartz-core/src/main/resources/org/quartz/impl/jdbcjobstore/tables_h2.sql

```diff
@@ -77,8 +77,7 @@ CREATE TABLE QRTZ_SIMPLE_TRIGGERS (
   TIMES_TRIGGERED BIGINT NOT NULL
 );
 
-CREATE TABLE qrtz_simprop_triggers
-  (          
+CREATE TABLE QRTZ_SIMPROP_TRIGGERS (          
```

Comment:
I understand the desire to align casing across the file, but could we keep the bug fix to just the single character change necessary to fix the bug.

Reply:
I am not completely sure that the different case between table definition and queries is not part of the issue with latest H2 1.4.200.
Hence, to stay safe, I changed that too.

Reply:
Unquoted names are case insensitive in H2 by default, but they are case-sensitive with some non-default settings. It's safer to use the same case of letters everywhere.

Reply:
Okay, I'll bow to the peer pressure.